.gitignore

```diff
@@ -0,0 +1,2 @@
+# dependencies
+/node_modules
```

Comment:
Is this working as expected? You can try creating template/node_modules folder and putting a dummy file inside (then `git status` will show if it's ignored).
Since it starts with `/`, it may be only ignoring node_modules folder in this level (root).
If that's the case, removing `/` should work.

Reply:
I have created a dummy file inside  `template/node_modules` folder. The status of the branch:
```
Your branch is up to date with 'origin/feat/protected-route'.
nothing to commit, working tree clean
```
I believe it is working as expected 👍🏾